openpype/hosts/blender/utility_scripts/update_representations.py

```diff
@@ -0,0 +1,107 @@
+import argparse
+import sys
+
+import bpy
+from openpype.client.entities import (
+    get_asset_by_name,
+    get_representation_by_id,
+    get_subset_by_name,
+)
+from openpype.hosts.blender.api.pipeline import (
+    AVALON_PROPERTY,
+    metadata_update,
+)
+from openpype.pipeline import legacy_io
+from openpype.pipeline.constants import AVALON_CONTAINER_ID
+
+if __name__ == "__main__":
+    # Parse arguments
+    parser = argparse.ArgumentParser(
+        description="Update AVALON metadata with given representation ID."
+    )
+    parser.add_argument(
+        "subset_name",
+        type=str,
+        nargs="?",
+        help="subset name",
+    )
+    parser.add_argument(
+        "--datablocks",
+        type=str,
+        nargs="*",
+        help="list of datablocks names to add container metadata to",
+        required=True,
+    )
+    parser.add_argument(
+        "--datapaths",
+        type=str,
+        nargs="*",
+        help="list of datapaths to get datablocks from",
+        required=True,
+        # TODO set default with outliner types
+    )
+    parser.add_argument(
+        "--id",
+        dest="representation_id",
+        type=str,
+        nargs="?",
+        help="representation ID",
+        required=True,
+    )
+    args, unknown = parser.parse_known_args(sys.argv[sys.argv.index("--") + 1 :])
+
+    containerized_datablocks = set()
+    for datapath in args.datapaths:
+        for datablock_name in args.datablocks:
+            datablock = getattr(bpy.data, datapath).get(datablock_name)
+            if (
+                not datablock
+                or datablock in containerized_datablocks
+                or datablock.get(AVALON_PROPERTY, {}).get("representation")
+            ):
+                continue
+
+            # Get docs
+            project_name = legacy_io.Session["AVALON_PROJECT"]
```

Comment:
I would expect these are passed to arguments. If not then please replace usage of `legacy_io` with functions from `openpype.pipeline`.